Automattic/wp-calypso
# Trail Map: Remove the hook that controls the language offering test

Related to #89692 

## Proposed Changes

This PR removes the `use-experiment-for-trail-map` hook and all the conditions bound to its returned values.

## Why are these changes being made?


Per announcement made in pau2Xa-5Ol-p2#comment-15735, the trail map language offering test has been concluded as rolling out the control variation. This PR is the 1st among the series other clean-up to come.

## Testing Instructions



* Mainly code review to make sure the clean-up makes sense. 
* The logged-in /plans and /start/plans should look and work as it used to be.

## Pre-merge Checklist
